util/topsql/stmtstats/aggregator.go

```diff
@@ -120,6 +117,7 @@ func (m *aggregator) unregisterCollector(collector Collector) {
 // close ends the execution of the current aggregator.
 func (m *aggregator) close() {
 	m.cancel()
+	m.running.Store(false)
```

Comment:
Why moving to here?

Reply:
Because in the original logic, After I call `m.close()` then call `m.start()`, the aggregator won't start again since `m.running` is still true in the original logic.

Reply:
We can wait for the goroutine to be done via `WaitGroup` in `close()`

Reply:
ok, done.